tooling-tests/src/test/resources/test-src/generated-sources/tool_test_data_type_22/duplicate_output_type_pb.bal

```diff
@@ -1,6 +1,6 @@
 import ballerina/grpc;
-import ballerina/protobuf.types.empty;
 import ballerina/protobuf.types.wrappers;
+import ballerina/protobuf.types.empty;
```

Comment:
the previous import order is correct. imports are in alphabetical order

Reply:
Ordered imports alphabetically in [340f63c](https://github.com/ballerina-platform/module-ballerina-grpc/pull/739/commits/340f63c0456580d9301e8e9c5fbb464efe1db910), [c5966e2](https://github.com/ballerina-platform/module-ballerina-grpc/pull/739/commits/c5966e20b3f4be74c504db145908be42caa3a236).